src/pages/community.tsx

```diff
@@ -108,148 +113,188 @@ const CommunityExpert: FunctionComponent<{ expert: typeof expertsData[0] }> = ({
+                            <button type="button" className="btn btn-primary my-8" onClick={() => setPage(page + 1)}>
+                                Load more
+                            </button>
+                        </div>
+                    )}
                 </div>
-                <Tabs
-                    tabs={[
-                        {
-                            key: 'upcoming_events',
-                            title: 'Upcoming',
-                            content: (
-                                <ul className="ml-0 list-outside list-none text-center md:text-left">
-                                    {events.upcoming.map((event, index) => (
-                                        <li key={index} className="p-[25px]">
-                                            <Heading size="h6">{event.date}</Heading>
-                                            <Heading size="h4">{event.title}</Heading>
-                                            <p className="mb-0 text-base tracking-[-0.25px]">{event.location}</p>
-                                        </li>
-                                    ))}
-                                </ul>
-                            ),
-                            className: 'pt-0 w-full text-2xl leading-[34px] pb-4',
-                        },
-                        {
-                            key: 'past_events',
-                            title: 'Past',
-                            content: (
-                                <ul className="ml-0 list-outside list-none text-center md:text-left">
-                                    {events.past.map((event, index) => (
-                                        <li key={index} className="p-[25px]">
-                                            <Heading size="h6">{event.date}</Heading>
-                                            <Heading size="h4">{event.title}</Heading>
-                                            <p className="mb-0 text-base tracking-[-0.25px]">{event.location}</p>
-                                        </li>
-                                    ))}
-                                </ul>
-                            ),
-                            className: 'pt-0 w-full text-2xl leading-[34px] pb-4',
-                        },
-                    ]}
-                    tabsWrapperClassName="md:w-[548px] max-w-[548px] bg-white border px-6 py-9 bg-gradient-to-b from-white from-7.29% to-gray-50 to-88.08% rounded-lg"
-                    contentClassName="pt-9 pb-0"
-                />
-            </div>
-        </ContentSection>
-    </Layout>
-)
+            </ContentSection>
+        </Layout>
+    )
+}
 
 export default Community
+
+export const getStaticProps: GetStaticProps<{
+    allPosts: Post[]
+    posts: Post[]
+}> = async ({ preview = false }) => {
+    const publishedPosts = await getAllPublishedBlogPosts('blog') // Replace this with guest-post
```

Comment:
```suggestion
    const publishedPosts = await getAllPublishedBlogPosts('guest-post') 
```

Commit this suggestion to pull blogs with `guest-post`.
I have used `blog` for demo purpose since `guest-post` has no blog yet.